src/api/__tests__/Annotations.test.js

```diff
@@ -175,6 +208,24 @@ describe('api/Annotations', () => {
                 errorCallback,
                 successCallback,
                 url: 'https://api.box.com/2.0/undoc/annotations/abc',
+                requestData: undefined,
```

Comment:
Is this expected?

Reply:
It is, as I did [extract requestData to a variable](https://github.com/box/box-ui-elements/pull/2982#discussion_r948145772). 
Another approach would be use previous notation for requestData in this method, to not include it in params at all. 
`...(shouldFetchReplies ? { requestData: { params: { fields: 'replies' } } } : null),`

Reply:
It is - I did extract [requestData to a variable ](https://github.com/box/box-ui-elements/pull/2982#discussion_r948145772). 
Another way would be to change notation to previous one, like: 
`...(shouldFetchReplies ? { requestData: { params: fields: 'replies' } } } : null),`
to not include requestData in get method at all